test/validation/api/ipsec/ipsec.c

```diff
@@ -1257,6 +1285,11 @@ int ipsec_term(odp_instance_t inst)
 			fprintf(stderr, "IPsec outq destroy failed.\n");
 	}
 
+	if (ODP_QUEUE_INVALID != default_queue) {
```

Comment:
In sync mode default_queue is not necessarily initialized to ODP_QUEUE_INVALID, depending on whether the macro maps to zeroes in the the tested ODP implementation or not.

Thus explicit initialization is needed. For the default queue it is done at the start of ipsec_init() (was added in e13682688).